manuscripts2/report.py

```diff
@@ -55,23 +64,23 @@ class Report():
 
     # Helper dict to map a data source class with its Elasticsearch index
     class2index = {
-        git.GitMetrics: GIT_INDEX,
-        github_issues.IssuesMetrics: GITHUB_ISSUES_INDEX,
-        github_prs.PullRequestsMetrics: GITHUB_PRS_INDEX,
+        git: GIT_INDEX,
+        github_issues: GITHUB_ISSUES_INDEX,
+        github_prs: GITHUB_PRS_INDEX
     }
 
     # Helper dict to map a data source name with its python class
     ds2class = {
-        "git": git.GitMetrics,
-        "github_issues": github_issues.IssuesMetrics,
-        "github_prs": github_prs.PullRequestsMetrics,
+        "git": git,
```

Comment:
Why you're not using the contants you defined earlier here? (I mean, `GIT_INDEX`, etc.).

Or even better, since this seems to be just a transposing of the previous dictionary, why not compute it instead to write it again "by hand"=

Reply:
Right, thanks! I'll change it.

Reply:
Thanks!